src/HomePage/components/MovieCard/MovieCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import Popup from "../Popup/Popup";
+import './MovieCard.css'
+import '../Popup/Popup.css'
+
+interface MovieCardInterface {
+    title: string,
+    popularity: number
+}
+
+export function MovieCard(props: MovieCardInterface) {
+    const [isOpen, setIsOpen] = useState(false);
+    const [isEdited, setIsEdited] = useState(false);
+    const [isDeleted, setIsDeleted] = useState(false);
+
+    const togglePopup = () => {
+        setIsOpen(!isOpen);
+    }
+
+    const toggleEdit = () => {
+        setIsEdited(!isEdited);
```

Comment:
Please remember of state asynchronity for cases when we need new state based on previous.
```
setIsEdited(current=>!current);
```